### DISCUSSION TITLE
Easier DateTime initialization

### DISCUSSION THREAD

**Discussion by pedrolamas** [dotnet/csharplang#6092] [Language Ideas] [answered]
Right now, the only way to initialize a `DateTime` in c# is:

```csharp
DateTime datetime1 = new DateTime(2022, 11, 1);
```

But in VB, we do have an alternative notation:

```vb
Dim date1 As Date = #5/1/2008 8:30:52AM#
```

So why not have something smarter in c# that would allow us to do similar things, like:

```text
DateTime datetime1 = #2022/11/1#; // compiler would translate to `new DateTime(2022, 11, 1)`

DateTime datetime1 = #2022/11/1 8:30:52AM#; // compiler would translate to `new DateTime(2022, 11, 1, 8, 30, 52)`
```

**Comment by ufcpp** [comment, marked as answer]
https://github.com/dotnet/csharplang/discussions/1969

**Reply by pedrolamas** [reply]
Thank you, wasn't aware that someone else already posted this! 🙂
